src/database.cc

```diff
@@ -2461,7 +2461,7 @@ error Database::saveModel(
                       "insert into autotracker_settings(uid, term, pid, tid) "
                       "values(:uid, :term, :pid, :tid)",
                       useRef(model->UID()),
-                      useRef(model->Term()),
+                      useRef(model->TermsString()),
```

Comment:
Using `useRef` with a r-value (that is a value - not reference - returned from a function) is dangerous. I think in this case there should be just plain getter call without the `useRef` wrapper. Same applies above in the update.

Reply:
An alternative would be just storing the delimited string. I see the API is mostly consumed with it anyway.

Reply:
I'd prefer to keep a vector of strings in memory, as we're optimizing for terms matching because it can be called much more frequently than the API methods, even though parsing probably won't be slow as users aren't expected to add lots of titles per single rule.
I'll check the Poco::Data docs first so I can better understand potential issues with `useRef` or other ways to do things.

Reply:
I totally get that. A completely alternative solution, while introducing C++17 (that will get to TogglDesktop in near future anyway) would be to use `std::string_view` and create a "view" over the delimited single string. This avoids splitting and copying parts of this string when we want to do comparisons. 

However, that'd increase the complexity of this PR a lot and would be a considerable time sink on your part, I assume.

Anyway, as far as I remember, having just `model->TermsString()` in the mentioned part of code (without wrapping it in `useRef`) should be enough. But it's definitely a good idea to consult the docs first.

Reply:
One other way occured to me when taking a shower: You could keep the code as is, but have a `static thread_local std::string` variable inside `TermsString` and return a reference to it. That way you're safe.

Reply:
Looks like `bind()` is recommended to use in this scenario: https://pocoproject.org/docs/00200-DataUserManual.html
>It is possible to use bind() instead of use(). The bind() call will always create a copy of the supplied argument.

I think I like it a bit more `thread_local std::string` just to keep things simpler in the model class.

`string_view` looks useful, it would be cool to use it later on.

Reply:
Alrighty, sounds good. I'll wait for the CI build and merge if it passes.